### PR TITLE
Add fibonacci series in java

### PR DESCRIPTION
**Pull request by AasthaGoyalgit**

### CODE REVIEW

File: java_fibonacci.java.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package milestone6;
+
+import java.util.Scanner;
+
+public class FibonacciSeries {
```
Comment (fharookshaik) [inline java_fibonacci.java.java:5]: Main.java:5: error: class FibonacciSeries is public, should be declared in a file named FibonacciSeries.java
public class FibonacciSeries {
       ^
1 error

Try to improve the code by fixing the errors.

Happy Coding!!
Happy Contributing!!

**Comment by AasthaGoyalgit** [comment]
Now once u check it again i have made some changes.

File: Java_fibonacci.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+public class Java_fibonacci {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		 Scanner sc=new Scanner(System.in);
+         System.out.println("Enter a number of members in series: ");
+         int n=sc.nextInt();
+         int count=0;
+         int a=1,b=1;
```
Comment (fharookshaik) [inline Java_fibonacci.java:13]: I'm encountering the following error after executing the program. Try changing them and add a new line after printing each fibonacci term.

Enter a number of members in series: 100
1 1 2 4 8 16 32 64 128 256 512 1024 2048 4096 8192 16384 32768 65536 131072 262144 524288 1048576 2097152 4194304 8388608 16777216 33554432 67108864 134217728 268435456 536870912 1073741824 -2147483648 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 

Happy Coding!!
Happy Contributing!!

**Review by fharookshaik — CHANGES_REQUESTED** [review]
Changes Requested.

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- Java_fibonacci.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288781&bid=20460280)

File: Java_fibonacci.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+         System.out.println("Enter a number of members in series: ");
+         int n=sc.nextInt();
+         int count=0;
+      
+        BigInteger a=new BigInteger("0");
+        BigInteger b=new BigInteger("1");
+        System.out.print(a+"\n "+b+"\n ");
+        while(count<(n-2))
+         {
+            BigInteger c=new BigInteger(a.add(b).toString());;
```
Comment (fharookshaik) [inline Java_fibonacci.java:19]: Codacy found an issue: [An empty statement (semicolon) not part of a loop](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288781)

File: Java_fibonacci.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package milestone6;
+
+import java.util.Scanner;
+
+public class Java_fibonacci {
```
Comment (fharookshaik) [inline Java_fibonacci.java:5]: Codacy found an issue: [The class name 'Java_fibonacci' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288781)

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- Java_fibonacci.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288781&bid=20460280)

File: Java_fibonacci.java (hunk 4 above)
Comment (fharookshaik) [inline Java_fibonacci.java:5]: Codacy found an issue: [The class name 'Java_fibonacci' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288781)

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Java_fibonacci.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288781&bid=20460280)

File: Java_fibonacci.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+         System.out.println("Enter a number of members in series: ");
+         int n=sc.nextInt();
+         int count=0;
+      
+        BigInteger a=new BigInteger("0");
+        BigInteger b=new BigInteger("1");
+        System.out.print(a+"\n "+b+"\n ");
+        while(count<(n-2))
+         {
+            BigInteger c=new BigInteger(a.add(b).toString());
```
Comment (fharookshaik) [inline Java_fibonacci.java:19]: Error while executing the code.


$javac milestone6/JavaFibonacci.java
milestone6/JavaFibonacci.java:14: error: cannot find symbol
        BigInteger a=new BigInteger("0");
        ^
  symbol:   class BigInteger
  location: class JavaFibonacci
milestone6/JavaFibonacci.java:14: error: cannot find symbol
        BigInteger a=new BigInteger("0");
                         ^
  symbol:   class BigInteger
  location: class JavaFibonacci
milestone6/JavaFibonacci.java:15: error: cannot find symbol
        BigInteger b=new BigInteger("1");
        ^
  symbol:   class BigInteger
  location: class JavaFibonacci
milestone6/JavaFibonacci.java:15: error: cannot find symbol
        BigInteger b=new BigInteger("1");
                         ^
  symbol:   class BigInteger
  location: class JavaFibonacci
milestone6/JavaFibonacci.java:19: error: cannot find symbol
            BigInteger c=new BigInteger(a.add(b).toString());
            ^
  symbol:   class BigInteger
  location: class JavaFibonacci
milestone6/JavaFibonacci.java:19: error: cannot find symbol
            BigInteger c=new BigInteger(a.add(b).toString());
                             ^
  symbol:   class BigInteger
  location: class JavaFibonacci
6 errors

**Comment by fharookshaik** [comment]
Thank you for your Contribution!!

**Comment by fharookshaik** [comment]
Don't forget to add your name and the programming language under the specific folders.

**Pull request merged**